runtime/plugins/comment/comment.lua

```diff
@@ -1,4 +1,4 @@
-VERSION = "1.0.0"
+VERSION = "1.1.0"
```

Comment:
As discussed in https://github.com/zyedidia/micro/pull/3673#pullrequestreview-2649030163 there is no need to change this version number, since the plugin is builtin.